### ISSUE TITLE
Setting row heights

### ISSUE THREAD

**Issue by miketricking** [ManifestWebDesign/angular-gridster#465]
Not particularly an issue more of a question.

I'm trying to dynamically set the row height. The height on the Items are set to auto and for now I have it working so it will search inside the grid for all items, grab the heights of them and then set rowHeight to the size of the tallest item. However I would like to set each row to different heights. So I need to loop through the items in the current row getting the heights to find the largest to then set that row height. However the problem I'm having is the the items are not nested in rows themself so I'm not sure how I could loop through items that are only showing in particular rows. 

`   var allElements = element[0].getElementsByClassName('gridster-item');

                    for (var i = 0; i < allElements.length; i++) {
                         scope.gridHeight.push(allElements[i].offsetHeight);
                    }

                    var largest = Math.max.apply(Math, scope.gridHeight);

                    scope.gridster.curRowHeight = (largest + 10);`

**Comment by samal84** [comment]
Yeah gridster items can span multiple rows, that's the conceptual mismatch here.

I guess if you don't have any items that span multiple rows in your layout, then you could manually loop over the elements and save them to an array using their row number as index. Then set each items height to the max of that row.
